core/src/main/java/org/geysermc/geyser/registry/populator/ItemRegistryPopulator.java

```diff
@@ -664,4 +620,29 @@ private static void registerFurnaceMinecart(int nextFreeBedrockId, List<Componen
         builder.putCompound("components", componentBuilder.build());
         componentItemData.add(new ComponentItemData("geysermc:furnace_minecart", builder.build()));
     }
+
+    /**
+     * Backwards-maps the items of 1.19.50 (630) to 1.19.40 (622)
```

Comment:
1.20.50/1.20.40 :p

Reply:
Not sure what you mean - this is mapping 1.19.50 block entries to 1.19.40 block entries - the raw bedrock nbt